.sun-ci.yml

```
@@ -0,0 +1,73 @@
+workspace: true
+
+stages:
+  - build
+  - test
+
+jobs:
+- name: build
+  stage: build
+  image: sunasteriskrnd/php-workspace:7.4
+  services:
+    - image: mysql:5.7
+      name: mysql_test
+      environment:
+         MYSQL_DATABASE: mysql
+         MYSQL_USER: user_test
+         MYSQL_PASSWORD: password_test
+         MYSQL_ROOT_PASSWORD: root
```

```suggestion
      name: mysql-test
      environment:
         MYSQL_DATABASE: mysql
         MYSQL_USER: user-test
         MYSQL_PASSWORD: password-test
         MYSQL_ROOT_PASSWORD: root
```
sửa lại chỗ này cho m nha... CI đổi tên service.

---

Dạ e đã sửa ạ